test/elixir_analyzer/exercise_test/common_checks/private_helper_functions_test.exs

```diff
@@ -0,0 +1,257 @@
+defmodule ElixirAnalyzer.ExerciseTest.CommonChecks.PrivateHelperFunctions.PacmanRulesTest do
+  use ElixirAnalyzer.ExerciseTestCase,
+    exercise_test_module: ElixirAnalyzer.TestSuite.PacmanRules
```

Comment:
Took me a while to realize there are two test modules in here. I don't see that often (if ever). I'm also missing more assertions about the comments (type, params). Could this test file maybe be rewritten into a single `ElixirAnalyzer.ExerciseTest.CommonChecks.PrivateHelperFunctionsTest` module that doesn't use `ElixirAnalyzer.ExerciseTestCase` but instead runs `ElixirAnalyzer.ExerciseTest.CommonChecks.PrivateHelperFunctions.run` directly and checks the whole comment struct? Like `ElixirAnalyzer.ExerciseTest.CommonChecks.ModulePascalCaseTest` for example.

Reply:
I'll try :)